src/python/pants/testutil/BUILD

```diff
@@ -29,6 +29,7 @@ python_library(
   sources = ['pants_run_integration_test.py'],
   dependencies = [
     '//:build_root',
+    ':process_handler',
```

Comment:
No need for this. It will get inferred. We only need `//:build_root` because it's a `files` target, and `src/python/pants:entry_point` because it's never explicitly imported.

You can confirm it shows up via `./pants dependencies src/python/pants/testutil:int-test`.

Reply:
Right - that just landed. Cool!